docs/documentation/integrations.md

```diff
@@ -139,7 +140,7 @@ Here are a few common macros used to document Zulip's integrations:
 [GitHub]: https://zulip.com/integrations/doc/github
 [codebase]: https://zulip.com/integrations/doc/codebase
 [beanstalk]: https://zulip.com/integrations/doc/beanstalk
-[1]: https://github.com/zulip/zulip/blob/708f3a4bb19c8e823c9ea1e577d360ac4229b199/zerver/lib/integrations.py#L78
+[integrations-file]: https://github.com/zulip/zulip/blob/708f3a4bb19c8e823c9ea1e577d360ac4229b199/zerver/lib/integrations.py
```

Comment:
This one has a specific commit hash, so shouldn't end up out-of-date; but I think it's better to just link to the file anyway.  Change the link to be on master rather than a specific commit hash?